docs/examples/stationary.py

```diff
@@ -0,0 +1,295 @@
+#
+# Because we use the `steps` suffix for these dimensions, Calliope will read them in as timestamps.
+
+# %%
+model.inputs.investsteps
+
+# %%
+model.inputs.vintagesteps
+
+# %% [markdown]
+# We can see some of our input data is indexed over these dimensions:
+
+# %%
+# Decreasing costs of investing in technologies
+model.inputs.cost_flow_cap.to_series().dropna()
+
+# %% [markdown]
+# ### Initial capacity
+#
+# Unlike greenfield optimisation ("plan" mode in Calliope), pathway optimisation should be initialised with a certain amount of existing technology capacity:
+
+# %%
+model.inputs.flow_cap_initial.to_series().dropna()
+
+# %% [markdown]
+# This technology capacity can then be phased out as we step through to the end of our time horizon:
+
+# %%
+# This is a fraction of the initial capacity that remains available in investment steps
+model.inputs.available_initial_cap.to_series().dropna()
+
+# %% [markdown]
+# ### Vintages
+#
+# Technology vintages have specific characteristics, such as the cost to invest in the model.
+# Recently commercialised technologies may see their deployment costs decrease substantially in the first decade or two of mass deployment.
+# Even established technologies can reduce in cost over time as the manufacturing facilities and logistical pipelines are constantly optimised.
+
+# %% [markdown]
+# End-of-life decommissioning is tracked with a matrix similar to [initial capacities](#initial-capacity).
+# Fractional availability accounts for technologies whose lifetimes fall in-between two investment steps.
+
+# Note how vintages are never available in investsteps that are in their _future_.
+model.inputs.available_vintages.to_series().dropna().unstack("investsteps")
+
+# %% [markdown]
+# ## Building a pathways optimisation problem
+#
+# We have created a math YAML file with updates to all the pre-defined math to handle the existence of `investsteps` and `vintagesteps`.
+# Tracking new capacity in each investment period and linking it to technology vintages requires new variables and constraints
+
+# %% [markdown]
+# ### Variables
+
+# %%
+# Note the "investsteps" dimension added to this pre-defined variable
+model.math["variables"]["flow_cap"]
+
+# %%
+# This new variable tracks the amount of each technology vintage that exists
+model.math["variables"]["flow_cap_new"]
+
+# %% [markdown]
+# ### Constraints
+
+# %%
+# All existing constraints have the "investsteps" dimension added.
+# This allows the dispatch decisions to be optimised individually for each investment step.
+model.math["constraints"]["system_balance"]
+
+# %%
+# In each investment period, capacities are a combination of all available vintages.
+model.math["constraints"]["flow_cap_bounding"]
+
+# %% [markdown]
+# ### Building
+
+# %%
+model.inputs.flow_cap_max.to_series().dropna()
+
+# %%
+model.build()
+model.backend.verbose_strings()
+model.backend.to_lp(OUTPUT_PATH+"test.lp")
```

Comment:
Looks like this is leftover from your own debugging

Reply:
removed!